pkg/apis/core/validation/validation_test.go

```diff
@@ -16500,12 +16500,12 @@ func TestIsValidSysctlName(t *testing.T) {
 		}(256),
 	}
 	for _, s := range valid {
-		if !IsValidSysctlName(s) {
+		if !IsValidSysctlName(s, false) {
```

Comment:
would be nice to check with both - `true` and `false`. This will validate that when allow slashes, all the old tests will keep running sucessfully.

Reply:
Thanks, Updated.